### DISCUSSION TITLE
Is it possible to install torch geometric with amd gpu

### DISCUSSION THREAD

**Discussion by vlashada** [pyg-team/pytorch_geometric#5612] [General]
The title basically sums it up. I have an AMD GPU, installed with `pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/rocm5.1.1`, as specified by [torch install](https://pytorch.org/get-started/locally/).
I know there was a question about this a year ago, but there were not answers.

**Comment by rusty1s** [comment]
It is now  possible to install `torch-scatter` for AMD GPUs from master due to https://github.com/rusty1s/pytorch_scatter/pull/325. Can you give it a try? There still might be issues with installing `torch-sparse` though.
